lib/repl.js

```diff
@@ -215,11 +215,16 @@ function REPLServer(prompt,
   }
 
   function defaultEval(code, context, file, cb) {
-    var err, result, script, wrappedErr;
+    var err, result, script, wrappedErr, trimCmd;
+    var isExitCmd = false;
     var wrappedCmd = false;
     var awaitPromise = false;
     var input = code;
 
+    if ((trimCmd = code.trim()) && (trimCmd === 'exit' || trimCmd === 'quit')) {
```

Comment:
Just a tiny style thing, but could you maybe turn `trimCmd = code.trim()` into its own line? Also, don’t be shy to use a verbose name like `trimCommand`/`isExitCommand`.

Reply:
Thank you soooo much about your advice! I will modify the code as soon.